modules/ipi-install-removing-the-provisioner-node.adoc

```diff
@@ -0,0 +1,13 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+
+[id="ipi-install-removing-the-provisioner-node_{context}"]
+= Removing the provisioner node
+
+Optionally, the provisioner node can be removed after installation.
+Verify a back up of the following:
+
+. The `install-config.yaml` file in case the cluster must be redeployed
```

Comment:
This is missing the .Procedure heading

And though it has steps, it also doesn't quite read as a procedure. I'd make these steps actionable, full sentences with a period